### PR TITLE
Use new operator SDK & k8s dependencies

### PR DESCRIPTION
**Pull request by der-eismann**

### CODE REVIEW

File: .egal/example.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
+/*
```
Comment (bjoernhaeuser) [inline .egal/example.go:1]: Should this file be removed?

**Review by bjoernhaeuser — APPROVED** [review]
LGTM, no idea :D

File: .egal/example.go (hunk 1 above)
Comment (der-eismann) [reply .egal/example.go:1]: oopsie, yes.

**Review by svenwltr — APPROVED** [review]
I totally read everything and it looks good to me.

**Pull request merged**